README.md

```diff
@@ -250,4 +250,57 @@ POST
     - if course is added successfully
         - `{"success": true, "message": "New Course successfully saved."}`
     - if course could not be added
-        - `{"success": false, "msg": "Unexpected error."}`
\ No newline at end of file
+        - `{"success": false, "msg": "Unexpected error."}`
+
+**/updateTeams**  
+
+POST
+- When the user is logged in and selects 'save teams' invoke gets data from new applicant form and passes into insertNewApplicantToDb function for inserting into database.
```

Comment:
A short and clear explanation of what the endpoint is for (updating a team on the database from form information?), rather than all this detail of what it does, would be much better as the first line.